zproject/settings.py

```diff
@@ -1043,6 +1043,10 @@ def get_secret(key):
 # Used for test_logging_handlers
 LOGGING_NOT_DISABLED = True
 
+# Enable to include the Python module where a log message comes from.
+# Useful for tracking down a mysterious log message, but a little verbose.
+LOGGING_SHOW_MODULE = False
+
```

Comment:
Should we put this into DEFAULT_SETTINGS, so it's easy to override on a production server without code changes?
  See https://zulip.readthedocs.io/en/latest/settings.html if you're not familiar with that system.

`LOGGING_NOT_DISABLED` is different, in that it's only ever changed in unit tests, not something you'd use in production.

Reply:
Oh, yeah, thanks -- this should definitely be made easy to override on a production server.

I'll go read that doc. :-) I think I'd noticed at some point that code in `settings.py` overrode a setting I tried to set in the config file, but had never dug into it.

Reply:
Sounds good.  I think it's also discussed briefly in the comments at the top of `settings.py`.  It may be worth doing a bit of work on making this more discoverable / the top of that doc more focused on "what to do" and less on the concepts.